src/app/zap-templates/zcl/data-model/chip/access-control-cluster.xml

```diff
@@ -14,11 +14,17 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 -->
-<configurator>
+<!--
+XML generated by Alchemy; DO NOT EDIT.
+Source: src/data_model/ACL-Cluster.adoc
+Parameters: in-progress 
+Git: 0.9-fall2024-87-gcdfdb86be-dirty
+-->
+<configurator xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../../zcl.xsd">
   <domain name="CHIP"/>
-
   <enum name="AccessControlEntryPrivilegeEnum" type="enum8">
     <cluster code="0x001F"/>
+    <cluster code="0x0752"/>
```

Comment:
The spec here is invalid, in referencing a data type from one cluster in another cluster.  The spec needs to be fixed.

More importantly, this is NOT just an "add conformance data" change.

Reply:
The point is: we should not be adding new XMLs as part of this automated PR.  Adding an XML file should be done when we are actually working on implementation for that cluster...